auction-app-backend/src/main/java/com/example/auctionapp/service/implementation/ProductServiceImpl.java

```diff
@@ -58,11 +58,23 @@ public ProductServiceImpl(final ProductRepository productRepository,
     }
 
     @Override
-    public ProductSearchResponse getProducts(UUID categoryId, String searchProduct, int page, int size) {
-        Pageable pageable = PageRequest.of(page, size);
+    public ProductSearchResponse getProducts(final UUID categoryId,
+                                             final String searchProduct,
+                                             final String sortField,
+                                             final String sortDirection,
+                                             final int page,
+                                             final int size) {
+        Sort sort = Sort.by(Sort.Direction.ASC, "name");  // default alphabetical sorting
```

Comment:
Why are we doing this here, why not in the Specification class?

Reply:
i moved the default sort criteria in controller as default values, since i need the sort criteria for my pageable in the service method